experimental/dds/tree2/src/feature-libraries/forestSummarizer.ts

```diff
@@ -109,8 +117,8 @@ export class ForestSummarizer implements Summarizable {
 			const treeBufferString = bufferToString(treeBuffer, "utf8");
 			// TODO: this code is parsing data without an optional validator, this should be defined in a typebox schema as part of the
 			// forest summary format.
-			const fields = parse(treeBufferString) as [FieldKey, EncodedChunk][];
-
+			// const fields = parse(treeBufferString) as [FieldKey, EncodedChunk][];
+			const fields = this.codec.decode(JSON.parse(treeBufferString));
 			const allocator = idAllocatorFromMaxId();
 			const delta: [FieldKey, Delta.FieldChanges][] = fields.map(([fieldKey, content]) => {
```

Comment:
ideally more of the logic here would live in the codec. It's fine for now, but maybe leave a comment suggesting this in `forestSummarizerCodec.ts`?

Reply:
Added comment to `forestSummarizerCodec.ts`